ArduSub/joystick.cpp

```diff
@@ -150,6 +150,7 @@ void Sub::transform_manual_control_to_rc_override(int16_t x, int16_t y, int16_t
 
 void Sub::handle_jsbutton_press(uint8_t _button, bool shift, bool held)
 {
+    float angle_max = aparm.angle_max/10;  // convert from deg*100 to deg*10
```

Comment:
Is this *really* in decidegrees?!  That's not a unit we've really used in ArduPilot.

I mean, I can see that it's multiplying a centdegrees by 10, which would give decidegrees, but isn't this really "your maximum trim angle is 10% of your maximum angle *in degrees*

Reply:
```suggestion
    const float trim_angle_max = aparm.angle_max * 0.1;  // trim angle is 10% of the maximum angle (degrees)
```